experimental/packages/opentelemetry-sdk-metrics-base/src/export/MetricReader.ts

```diff
@@ -53,8 +54,8 @@ export abstract class MetricReader {
   /**
    * Get the {@link AggregationTemporality} preferred by this {@link MetricReader}
    */
-  getPreferredAggregationTemporality(): AggregationTemporality {
-    return this._preferredAggregationTemporality;
+  getAggregationTemporality(instrumentType: InstrumentType): AggregationTemporality {
```

Comment:
Since this function now receives the instrument type as arguments, I think it would be more intuitive to name it as `selectAggregationTemporality` or `aggregationTemporalityFor` to indicate that the result may differ in various conditions.

Reply:
renamed it to `selectAggregationTemporality`